video/out/vulkan/context_mac.m

```diff
@@ -128,6 +128,7 @@ static int mac_vk_control(struct ra_ctx *ctx, int *events, int request, void *ar
 const struct ra_ctx_fns ra_ctx_vulkan_mac = {
     .type           = "vulkan",
     .name           = "macvk",
+    .description    = "Vulkan on macOS (MoltenVK)",
```

Comment:
i wouldn't mention MoltenVK specifically. atm it's independent of MoltenVK and it can in theory use any vulkan driver/loader that could be available on macOS (and uses a metal layer). just that MoltenVK is the only one.

also since the naming scheme of the other contexts seems to be `<PLATFORM>/<API>`, it might be better to name it something like `mac/Vulkan (Metal)` or `mac/Vulkan (via Metal)`. mentioning metal since it's implemented via a Vulkan > Metal translation layer.